### PR TITLE
LBNL ISO queues zenodo store

### PR DESCRIPTION
**Pull request by bendnorman**

### CODE REVIEW

File: frictionless/lbnlisoqueues.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        "in concert with 35 non-ISO utilities, representing an estimated "
+        "85% of all U.S. electricity load. We include all 'active' projects "
+        "in these generation interconnection queues through the end of 2020, "
+        "as well as data on 'completed' and 'withdrawn' projects for five of "
+        "the ISOs (CAISO, ISO-NE, MISO, NYISO, PJM).",
+    "profile": "data-package",
+    "keywords": [
+        "lbln", "usa", "electricity", "iso", "renewables", "emissions", "utilities", "iso", "interconnection",
+    ],
+    "licenses": [licenses.us_govt, ],
```
Comment (zaneselvans) [inline frictionless/lbnlisoqueues.py:21]: Are you sure this is accurate? The national labs aren't technically part of the USG, and their products may or may not be subject to the constitutional copyright exclusion.

File: frictionless/lbnlisoqueues.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
+"""Provide datapackage details specific to the EIP Infrastructure archives."""
+
+from . import core
+from . import contributors
+from . import licenses
+
+lbnlisoqueues = {
+    "name": "raw-lbnlisoqueues",
+    "title": "Raw LBLN ISO Interconnection Queues",
```
Comment (zaneselvans) [inline frictionless/lbnlisoqueues.py:9]: Is it LBL (Lawrence Berkeley Lab) or LBNL (Lawrence Berkeley National Lab). I think these may be different things. And in any case LBLN seems like it's probably a typo

File: frictionless/lbnlisoqueues.py [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        "the ISOs (CAISO, ISO-NE, MISO, NYISO, PJM).",
+    "profile": "data-package",
+    "keywords": [
+        "lbln", "usa", "electricity", "iso", "renewables", "emissions", "utilities", "iso", "interconnection",
+    ],
+    "licenses": [licenses.us_govt, ],
+    "homepage": "https://catalyst.coop/pudl/",
+    "sources": [
+        {
+            "title": "LBLN ISO Interconnection Queues",
```
Comment (zaneselvans) [inline frictionless/lbnlisoqueues.py:25]: LBNL?

File: frictionless/lbnlisoqueues.py [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+        "85% of all U.S. electricity load. We include all 'active' projects "
+        "in these generation interconnection queues through the end of 2020, "
+        "as well as data on 'completed' and 'withdrawn' projects for five of "
+        "the ISOs (CAISO, ISO-NE, MISO, NYISO, PJM).",
+    "profile": "data-package",
+    "keywords": [
+        "lbln", "usa", "electricity", "iso", "renewables", "emissions", "utilities", "iso", "interconnection",
+    ],
+    "licenses": [licenses.us_govt, ],
+    "homepage": "https://catalyst.coop/pudl/",
```
Comment (zaneselvans) [inline frictionless/lbnlisoqueues.py:22]: Should this really point at PUDL? Or should it point to the page at LBNL?

File: frictionless/lbnlisoqueues.py [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+    Args:
+        dfiles: iterable of file descriptors, as expected from Zenodo.
+            https://developers.zenodo.org/#deposition-files
+
+    Returns:
+        dict: fields suited to the frictionless datapackage spec
+        https://frictionlessdata.io/specs/data-package/
+
+    """
+    return core.minimal_datapackager(lbnlisoqueues, dfiles)
```
Comment (zaneselvans) [inline frictionless/lbnlisoqueues.py:46]: Missing newline at EOF

File: zs/metadata.py [hunk 6]
```diff
@@ -265,4 +265,24 @@
     "keywords": [
         "eip", "usa", "electricity", "infrastructure", "fossil fuel", "emissions", "oil", "gas", "chemicals", "pipelines", eipinfrastructure_uuid
     ]
+}
+
+# LBLN ISO Queues
+lbnlisoqueues_uuid = "36f4c418-5d16-11ec-9a1d-acde48001122"
+lbnlisoqueues = {
+    "title": "LBLN ISO Queues",
```
Comment (zaneselvans) [inline zs/metadata.py:273]: LBLN?

File: zs/metadata.py [hunk 7]
```diff
@@ -265,4 +265,24 @@
...
+    "language": "eng",
+    "upload_type": "dataset",
+    "description": "<p>LBLN ISO Interconnection Queues "
```
Comment (zaneselvans) [inline zs/metadata.py:276]: LBLN?

File: zs/metadata.py [hunk 8]
```diff
@@ -265,4 +265,24 @@
...
+                   "https://emp.lbl.gov/publications/queued-characteristics-power-plants"
+                   "</a></p>"
+                   f"{pudl_description}",
+    "creators": creators,
+    "access_right": "open",
+    "license": "other-pd",
+    "keywords": [
+        "lbln", "usa", "electricity", "iso", "renewables", "emissions", "utilities", "iso", "interconnection", lbnlisoqueues_uuid
+    ]
 }
```
Comment (zaneselvans) [inline zs/metadata.py:288]: newline at EOF

File: zs/metadata.py [hunk 9]
```diff
@@ -265,4 +265,24 @@
...
+                   "archived from\n"
+                   "<a href=\"https://emp.lbl.gov/publications/queued-characteristics-power-plants\">"
+                   "https://emp.lbl.gov/publications/queued-characteristics-power-plants"
+                   "</a></p>"
+                   f"{pudl_description}",
+    "creators": creators,
+    "access_right": "open",
+    "license": "other-pd",
+    "keywords": [
+        "lbln", "usa", "electricity", "iso", "renewables", "emissions", "utilities", "iso", "interconnection", lbnlisoqueues_uuid
```
Comment (zaneselvans) [inline zs/metadata.py:286]: LBLN? ISO is in there twice.

**Review by zaneselvans — CHANGES_REQUESTED** [review]

File: frictionless/lbnlisoqueues.py (hunk 2 above)
Comment (bendnorman) [reply frictionless/lbnlisoqueues.py:9]: Their domain and website give mixed signals but I'm pretty sure it is [Lawrence Berkeley National Lab](https://en.wikipedia.org/wiki/Lawrence_Berkeley_National_Laboratory).

File: frictionless/lbnlisoqueues.py (hunk 1 above)
Comment (bendnorman) [reply frictionless/lbnlisoqueues.py:21]: I'm not sure. I couldn't find a definitive license for LBNL data. I might need to email them.

Comment (bendnorman) [reply frictionless/lbnlisoqueues.py:21]: Same goes for the EIP dataset.

**Comment by bendnorman** [comment]
We are now just storing the raw data in the down-ballot-climate-partners/deployment-gap-model repo. The sanbox archives of these datasets have been removed from zenodo.

**Pull request closed without merging**